### PR TITLE
IGNITE-11780

### PR DESCRIPTION
**Pull request by EdShangGG**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/BaselineCommand.java [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+ */
+public class BaselineCommand implements Command<BaselineArguments> {
+    /** Arguments. */
+    private BaselineArguments baselineArgs;
+
+    /** {@inheritDoc} */
+    @Override public void printUsage(CommandLogger logger) {
+        final String constistIds = "consistentId1[,consistentId2,....,consistentIdN]";
+
+        Command.usage(logger,"Print cluster baseline topology:", BASELINE);
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/BaselineCommand.java]: Could you please reformat code? Command.usage(logger, (space here)

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/BaselineCommand.java [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+    /**
+     * Change baseline.
+     *
+     *
+     * @param clientCfg Client configuration.
+     * @throws Exception If failed to execute baseline action.
+     */
+    @Override public Object execute(GridClientConfiguration clientCfg, CommandLogger logger) throws Exception {
+        try (GridClient client = Command.startClient(clientCfg)){
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/BaselineCommand.java]: Maybe we need to add a space after  try(), need we?

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/DeactivateCommand.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    /**
+     * Deactivate cluster.
+     *
+     * @param clientCfg Client configuration.
+     * @throws Exception If failed to deactivate.
+     */
+    @Override public Object execute(GridClientConfiguration clientCfg, CommandLogger logger) throws Exception {
+        try (GridClient client = Command.startClient(clientCfg)){
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/DeactivateCommand.java]: I suggest to format as ')) {'

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/TxCommands.java [hunk 4]
```diff
@@ -0,0 +1,582 @@
...
+
+    /** Arguments */
+    private VisorTxTaskArg args;
+
+    /** Logger. */
+    private CommandLogger logger;
+
+    /** {@inheritDoc} */
+    @Override public void printUsage(CommandLogger logger) {
+        Command.usage(logger,"List or kill transactions:", TX, getTxOptions());
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/TxCommands.java]: I suggest adding space after logger,

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+import static org.apache.ignite.internal.commandline.CommonArgParser.getCommonOptions;
+import static org.apache.ignite.internal.commandline.cache.CacheSubcommands.CONTENTION;
+import static org.apache.ignite.internal.commandline.cache.CacheSubcommands.FIND_AND_DELETE_GARBAGE;
+import static org.apache.ignite.internal.commandline.cache.CacheSubcommands.HELP;
+import static org.apache.ignite.internal.commandline.cache.CacheSubcommands.LIST;
+import static org.apache.ignite.internal.commandline.cache.CacheSubcommands.VALIDATE_INDEXES;
+import static org.apache.ignite.spi.discovery.tcp.ipfinder.sharedfs.TcpDiscoverySharedFsIpFinder.DELIM;
+
+/**
+ * High-level "cache" command realization.
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java]: Was 'implementation' meant by 'realization'?

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+        });
+
+        logger.nl();
+    }
+
+
+    /**
+     * Print cache command usage with default indention.
+     *
+     * @param cmd Cache command.
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java]: logger parameter missed in Javadoc

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+            usageCacheParams(logger, paramsDesc, indentsNum + 1);
+
+            logger.nl();
+        }
+    }
+
+    /**
+     * Print cache command arguments usage.
+     *
+     * @param paramsDesc Cache command arguments description.
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java]: logger parameter missed in javadoc

File: modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/IgniteCacheGroupsWithRestartsTest.java [hunk 8]
```diff
@@ -223,15 +223,15 @@ private void testFindAndDeleteGarbage(
...
     /**
      * @param ignite Ignite to execute task on.
-     * @param deleteFoundGarbage If clearing mode should be used.
+     * @param deleteFoundGarbarge If clearing mode should be used.
      * @return Result of task run.
      */
     private VisorFindAndDeleteGarbageInPersistenceTaskResult executeTask(
         IgniteEx ignite,
-        boolean deleteFoundGarbage
+        boolean deleteFoundGarbarge
```
Comment (dspavlov) [inline modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/IgniteCacheGroupsWithRestartsTest.java]: Is it a misprint here Garbarge ?

Comment (EdShangGG) [reply modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/IgniteCacheGroupsWithRestartsTest.java]: https://en.oxforddictionaries.com/thesaurus/garbage

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java (hunk 5 above)
Comment (EdShangGG) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheCommands.java]: Yes.

**Pull request closed without merging**